feature_layers/display-route-layer/README.md

```diff
@@ -0,0 +1,33 @@
+# Display route layer
+
+Display a route layer and its directions using feature collection.
+
+![Image of display map](DisplayRouteLayer.png)
+
+## Use case
+
+Routes can be stored as feature collection layers. These layers can store useful information such as directions, estimated trip time, and more.
+
+You can create a route layer in ArcGIS Pro and store a route layer as a portal item, making it easy to access, share, or display.
+## How to use the sample
+
+Pan and zoom to view the route displayed by the feature collection layer. Tap the toolbar button to view the list of directions.
```

Comment:
```suggestion
Pan and zoom to view the route displayed by the feature collection layer. The list of directions are displayed in the UI.
```
We need to alter this slightly to make sense for our app (on iOS their directions are hidden behind a button, but here we are just displaying ours)